src/main/java/com/example/WMS/services/servicesimpl/OrderServicesImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.example.WMS.services.servicesimpl;
+
+import com.example.WMS.dtos.AddStockDescriptionDto;
+import com.example.WMS.dtos.OrderDescriptionDto;
+import com.example.WMS.dtos.OrderDto;
+import com.example.WMS.dtos.ValidationDto;
+import com.example.WMS.entity.Order;
+import com.example.WMS.entity.ProductDetails;
+import com.example.WMS.handlers.OrderHandler;
+import com.example.WMS.handlers.ProductDetailsHandler;
+import com.example.WMS.handlers.ReadWriteHandler;
+import com.example.WMS.handlers.ReceiveOrdersHandler;
+import com.example.WMS.services.OrderServices;
+import org.springframework.stereotype.Service;
+
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class OrderServicesImpl implements OrderServices {
+    public ValidationDto createOrder(OrderDto orderDto) {
+
+        ProductDetails productDetails = ProductDetailsHandler.getProductDetails();
```

Comment:
Why have you made getProductDetails static and not a public method and make ProductDetailsHandler as component?

Reply:
When i started with the project i decided to handle threads on my own and run handlers on the thread, getting details was a static method i would call when the thread completed its execution, to get the values that the thread has got from the file. However, i ran into issues while using jackson object mapper so i discarded it and stopped using threads and forgot to change it back to a normal function.